packages/@okta/vuepress-site/docs/concepts/oauth-openid/index.md

```diff
@@ -28,19 +28,19 @@ OAuth 2.0 is a standard that apps use to provide client applications with access
 
 The OAuth 2.0 spec has four important roles:
 
-* The "authorization server" &mdash; The server that issues the access token. In this case Okta is the authorization server.
+* **authorization server**: The server that issues the access token. In this case Okta is the authorization server.
 
-* The "resource owner" &mdash; Normally your application's end user that grants permission to access the resource server with an access token
+* **resource owner**: Normally your application's end user that grants permission to access the resource server with an access token.
```

Comment:
```suggestion
* **resource owner**: The entity that grants permission to access the resource server with an access token (typically, this is the application's end user)
```
Making this statement parallel with the other role definitions?

Reply:
As discussed over Slack will stick with the period.